bebraven/platform
# Ignore Zoom meeting deleted error when cancelling registration

### Description
Task: https://app.asana.com/0/1174274412967132/1201095615146481

### Test Plan
1. Fully enroll and sync a Fellow Participant who is in a Cohort Schedule with a valid Zoom meeting ID set.
1. Make sure they get a Zoom link generated
2. Mark them as Dropped and run the sync
3. Make sure their Zoom meeting link is deleted in Salesforce
4. Mark them enrolled and sync again to get their link back
5. Delete the Zoom meeting, mark them as Dropped, and run the sync
6. Make sure the Zoom link is deleted and the sync email says success. No exceptions should bubble, but a `app.zoom.participant.skip_reason` Honeycomb field will be there indicating that the ZoomAPI call failed.

### Screenshots (if appropriate):


### Types of changes




### Checklist:
